task.sh

```diff
@@ -2,42 +2,58 @@
 
 # создаём каталог task с вложенными директориями
 # task
+mkdir ./task
```

Comment:
Можно сделать за одну команду
```suggestion
mkdir -p task/dir1 task/dir2 task/dir3/dir4
```